winton_kafka_streams/processor/_context.py

```diff
@@ -47,36 +49,36 @@ def schedule(self, timestamp):
 
         pass
 
-    @property
+    @property  # type: ignore
     @_raiseIfNullRecord
     def offset(self):
-        return self.currentRecord.offset()
+        return self.current_record.offset()
 
-    @property
+    @property  # type: ignore
     @_raiseIfNullRecord
     def partition(self):
-        return self.currentRecord.partition()
+        return self.current_record.partition()
 
-    @property
+    @property  # type: ignore
     @_raiseIfNullRecord
     def timestamp(self):
-        return self.currentRecord.timestamp()
+        return self.current_record.timestamp()
 
-    @property
+    @property  # type: ignore
```

Comment:
What are these ignores about?

Reply:
it's wating on https://github.com/python/mypy/issues/1362
Have put a comment in linking the issue